_data/collection-index.yml

```
@@ -783,3 +783,9 @@
   contact: https://github.com/nils-geistmann/devcontainers-features/issues
   repository: https://github.com/nils-geistmann/devcontainers-features/
   ociReference: ghcr.io/nils-geistmann/devcontainers-features
+- name: OpenFGA CLI
+  maintenance: Andrew Porter
+  contact: https://github.com/partydrone/devcontainer/issues
+  repository: https://github.com/partydrone/devcontainer
+  ociReference: ghcr.io/partydrone/devcontainer/features/openfga-cli
```

```suggestion
  ociReference: ghcr.io/partydrone/devcontainer/features
```

Hi 👋 

The `ociReference` here refers to the collection metadata artifact instead of the individual Features. Hence, the author needs to only add a single entry irrespective of the number of Features that might exist in their repository.

Can you help update this? thanks!

---

No problem! Thank you for clarifying. 🙂